Taskfile.yml

```diff
@@ -60,6 +65,7 @@ tasks:
           cp $FLAGS_FILE workspaces/cli-scripts/.env
           cp $FLAGS_FILE workspaces/optic-engine/.env
           cp $FLAGS_FILE workspaces/optic-engine-native/.env
+          cp $FLAGS_FILE workspaces/ui-v2/.env.development.local
```

Comment:
This should be fine to add in - this env variable is used by the create-react-app `yarn start` instance, the production env variable is used for the `yarn build` instance